src/metax_api/models/catalog_record.py

```diff
@@ -1007,13 +1036,19 @@ def _pre_update_operations(self):
         else:
             self.update_datacite = False
 
-        if self.catalog_versions_datasets() and not self.preserve_version:
+        if self.field_changed('cumulative_state'):
+            # there is a RPC api for changing the state
+            self.cumulative_state = self._initial_data['cumulative_state']
```

Comment:
I think we should raise a 400 here instead with a message "blabla use api /rpc/datasets/xyz to change cumulative state".
(It occurred to me that this rpc is not in line with changing preservation_state, which can be updated same way as all other fields... Maybe we should re-visit updating preservation_state in near future too...)